Source/Diagnostics/ReducedDiags/FieldProbeParticleContainer.H

```diff
@@ -19,13 +19,20 @@
  * This enumerated struct is used to index the field probe particle
  * values that are being stored as SoA data. Nattribs
  * is enumerated to give the number of attributes stored.
+ * The strange insertion of `theta` below is due to the use of
```

Comment:
This is fine for now, though a bit kludgy. Perhaps a fix is to add an optional argument to `GetParticlePosition` that specifies the index for theta (defaulting to `PIdx::theta`). That would be messy though since it would have to be inside of `ifdef WARPX_DIM_RZ` macros.